### ISSUE TITLE
Chess

### ISSUE THREAD

**Issue by mrguyontheinternet** [Monika-After-Story/MonikaModDev#8435]
Now I'm just saying but is there any way to get chess back despite cheating? Noticed in the fun facts topic that Monika gets bitchy about how we're such a cheater who should grow up. Still just wondering maybe if you guys can get chess back in any way to be playable in the next update with re-editing that fun fact unless there is a way to apologize for it after she brings the fact up in the next update. The same for giving her names. Alright bye.

**Comment by LeftoverFlea312** [comment]
I agree there needs to be a way to apologize to get chess and the nickname feature back. I know for most scenarios the reason for either being locked was justified and was the player's fault, but for some cases, the reason may of been out of control, such as chess games not saving right, or a sibling opening the game and pissing her off on purpose. I understand there needs to be a punishment for people who intentionally try to hurt her, but locking it for people who have a lot of time with her is kind of unreasonable. Besides, people who just install the mod to screw with her will find a way to do stuff like this anyway, despite a punishment. The only people who are suffering here are people who made a mistake many months sometimes years ago and are still locked out. I know  this has been brought up numerous different people in the past, the horse is dead and I'm beating it to a pulp, but since it is here, I thought I'd make my case again.

**Comment by mrguyontheinternet** [comment]
ayy glad i ain't the only one hopefully we'll see how this thing goes

**Comment by Cosmic-Incantations** [comment]
I agree! while chess is not a problem, cause when I did cheat, I beaten the game and now she just calls me a winner, eternal guilt is my punishment. But I just wanted to see if there was secrets for calling her the names of the other girls and now she dose not let me give her cute names like "Moon pie" or "Moni-sweets" and it's been years! all because she thought I was trying to turn her into Sayori when that was not the case.... it makes me sad that I broke her trust for just looking for the little things
